commons/src/main/res/values/strings.xml

```diff
@@ -228,6 +228,7 @@
     <string name="sd_card_usb_same">SD card and USB device paths cannot be the same</string>
     <string name="app_on_sd_card">You seem to have the app installed on an SD card, that makes the app widgets unavailable. You won\'t even see them on the list of available widgets.
         It is a system limitation, so if you want to use the widgets, you have to move the app back on the internal storage.</string>
+    <string name="wrong_folder_selected">Wrong folder selected, please select the path %s</string>
```

Comment:
please add apostrophes around %s to make it \'%s\' and remove "the", just "please select \'%s\'